packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -308,6 +308,7 @@ export default function AnimalInventory({
             photoUrl={d.photo_url}
             subtext={isDesktop ? null : `${d.type} / ${d.breed}`}
             highlightedText={d.batch ? d.count : null}
+            className={view === View.TASK_SUMMARY ? '' : styles.cellAfterCheckbox}
```

Comment:
Could you double check this condition? I'm not so confident!

Reply:
That should be right, now that the worker view was adjusted!